README.md

```diff
@@ -1,10 +1,35 @@
-# @crowdstrike/ember-toucan-core
+# Toucan Core
+
+<p align="center">
+  <a href="https://www.crowdstrike.com">
+    <img src="https://github.com/CrowdStrike/ember-toucan-core/blob/main/.github/cs-logo.png?raw=true" alt="CrowdStrike logo" width="300" />
+  </a>
+</p>
+
+<h1 align="center">The Toucan Design System from CrowdStrike</h1>
+
+<br>
 
 ![CI status](https://github.com/crowdstrike/ember-toucan-core/actions/workflows/ci.yml/badge.svg?branch=main)
 
-Toucan provides a set of accessible and reusable components that make it easy to create visually consistent and efficient Ember applications.
+<br />
+
+Toucan provides a set of accessible and reusable components that make it easy to create visually consistent and efficient Ember applications. This repository is a monorepo publishing two packages:
+
+- `ember-toucan-core`
+- `ember-toucan-form`
+
+The `core` package contains the Toucan-styled Ember components. The `form` package allows users to build forms using [`ember-headless-form`](https://github.com/CrowdStrike/ember-headless-form) with the `core` components.
+
+### Compatibility
+
+- Ember.js 4.8 or above
+- Embroider or ember-auto-import v2
```

Comment:
Just curious, what makes it `ember-auto-import` able? It being v2?

Reply:
@ynotdraw suggestion, nit, totally non-blocking: `v2` -> `2.0 or above` (since Ember auto-import is `2.6.3` now).

This might just be me, but I was mixing up `v2` with `v2 addons` not `version 2 plus of ember-auto-import`. 

>You can think of ember-auto-import as the "Embroider-polyfill for legacy apps"

😮 This is really helpful and should probably be somewhere in the README of ember-auto-import.  I think I've spent probably too much time wondering "what makes it auto-import if you have to manually write out the import still?"

Reply:
Done!